packages/create-cloudflare/turbo.json

```diff
@@ -1,11 +1,9 @@
 {
 	"$schema": "http://turbo.build/schema.json",
 	"extends": ["//"],
-
 	"pipeline": {
 		"build": {
 			"env": [
-				"VITEST",
```

Comment:
removing `VITEST` since this is declared as a `globalEnv` in https://github.com/cloudflare/workers-sdk/blob/main/turbo.json#L6